### PR TITLE
Update CS

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -24,56 +24,9 @@ if (PHP_VERSION_ID < 70000) {
...
-        'comment_to_phpdoc' => true,
-        'compact_nullable_typehint' => true,
-        'escape_implicit_backslashes' => true,
-        'explicit_indirect_variable' => true,
-        'explicit_string_variable' => true,
-        'final_internal_class' => true,
-        'fully_qualified_strict_types' => true,
-        'function_to_constant' => ['functions' => ['get_class', 'get_called_class', 'php_sapi_name', 'phpversion', 'pi']],
+        '@PhpCsFixer' => true,
+        '@PhpCsFixer:risky' => true,
```
Comment (keradus) [inline .php_cs.dist:28]: what about @PHPUnit60Migration:risky ?

File: .php_cs.dist [hunk 2]
```diff
@@ -24,56 +24,10 @@ if (PHP_VERSION_ID < 70000) {
 return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
-        '@PHP56Migration' => true,
```
Comment (keradus) [inline .php_cs.dist:27]: what about this ?

Comment (kubawerlos) [reply .php_cs.dist:27]: It has no rules: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.14.2/src/RuleSet.php#L289 - maybe `@PHP56Migration:risky` instead?

Comment (keradus) [reply .php_cs.dist:27]: Wow. Yes, please

**Pull request merged**